liferay-content-management/liferay-portal
# LPD-28040 Remove Feature Flag for Manage WC permissions in bulk

Remove Feature Flag for Manage WC permissions in bulk

# What is this trying to solve?

[Link to ticket](https://issues.liferay.com/browse/LPD-28040)

This PR removes the FF for https://liferay.atlassian.net/browse/LPD-16469

# How am I fixing it?

1. Remove the FF from the code
2. Remove the FF from portal.properties and language.properties

# How can you verify that it works?

Deploy the branch and check that the bulk change permissions action is available in web content admin
